.github/workflows/main.yml

```diff
@@ -151,7 +151,7 @@ jobs:
     - name: Test
       run: |
         echo "::add-matcher::./.github/workflows/main/problemMatchers/unittest.json"
-        ${{ matrix.testRunner }} "${{ env.GAFFER_BUILD_DIR }}/bin/gaffer test"
+        ${{ matrix.testRunner }} "${{ env.GAFFER_BUILD_DIR }}/bin/gaffer test -stopOnFailure"
```

Comment:
I think I would prefer to omit this. Better to know that lots of tests have failed and fix them all at once than go through several iterations of fix/push/failOnSomethingElse.

Reply:
Oh I thought you added it in on purpose before?